middleware.go

```diff
@@ -414,11 +413,7 @@ func (p *processorWithSchemaEncode) Specification() (Specification, error) {
 }
 
 func (p *processorWithSchemaEncode) Configure(ctx context.Context, config config.Config) error {
-	err := p.Processor.Configure(ctx, config)
-	if err != nil {
-		return err //nolint:wrapcheck // middleware shouldn't wrap errors
-	}
-
+	var err error
 	encodeKey := *p.defaults.KeyEnabled
 	if val, ok := config[configProcessorSchemaEncodeKeyEnabled]; ok {
 		encodeKey, err = strconv.ParseBool(val)
```

Comment:
We need to also delete  `configProcessorSchemaEncodeKeyEnabled` and `configProcessorSchemaEncodePayloadEnabled` in these two ifs.